languages/de-DE/Buttons.multids

```diff
@@ -30,12 +30,22 @@ ExportTiddler/Caption: Exportieren
 ExportTiddler/Hint: Diesen Tiddler Exportieren
 ExportTiddlers/Caption: Mehrere Exportieren
 ExportTiddlers/Hint: Mehrere Tiddler Exportieren
+Fold/Caption: Ausblenden Textbereich
```

Comment:
I know, but I'd like to get shorter terms and not longer. Those Captions can be used as "text only buttons" in the UI. So the texts should be as short as possible.